src/smart-components/user/users-list-not-selectable.tsx

```diff
@@ -43,6 +43,7 @@ const UsersListNotSelectable = ({ userLinks, usesMetaInURL, props }: UsersListNo
   const appNavigate = useAppNavigate(`/${getBundle()}/${getApp()}`);
   // use for text filter to focus
   const innerRef = useRef<HTMLInputElement | HTMLTextAreaElement>(null);
+  const advancedPermissions = useFlag('platform.rbac.common-auth-model_advanced-permissions');
```

Comment:
The advanced permission feature flag is for different feature. We want to use `platform.rbac.common-auth-model`
```suggestion
  const authModel = useFlag('platform.rbac.common-auth-model');
```

Reply:
Fixed